src/pet.js

```diff
@@ -0,0 +1,73 @@
+const max_fitness = 10;
+const empty_belly = 0;
+const grow_old = 1;
+
+function Pet (name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = empty_belly;
+    this.fitness = max_fitness;
+    this.child = [];
+    };
+
+    Pet.prototype = {
+        get isAlive(){
+            return this.age < 30 && this.hunger < 10 && this.fitness > 0
+        }
+    };
+    
+    Pet.prototype.growUp = function (){
+        if (!this.isAlive){
+            throw new Error('Your pet is no longer alive')
+        }
+        this.age += grow_old;
+    };
+    Pet.prototype.getHungry = function (){ 
+        this.hunger += 5;
+    };
+    Pet.prototype.becomeUnfit = function (){
+        this.fitness -= 3;
+    };
+    Pet.prototype.walk =  function (){
+        if (!this.isAlive){
+            throw new Error('Your pet is no longer alive')
+        }
+        if((this.fitness + 4) <= max_fitness){
+            this.fitness += 4
+        } else {
+            this.fitness = max_fitness
+        }
```

Comment:
This could be refactored a bit. In essence you are adding a value to the existing fitness, without allowing it go over a set limit. You can do this with Math.min, which will return the lowest of the values passed to it:
```
    this.fitness = Math.min(this.fitness + 4, MAX_FITNESS);
```